and-bible/app/src/main/java/net/bible/android/control/page/window/WindowControl.java

```diff
@@ -300,7 +300,7 @@ public void onEvent(CurrentVerseChangedEvent event) {
 		windowSync.synchronizeScreens();
 	}
 
-	public void onEvent(BookmarkChangedEvent event) {
+	public void onEvent(SynchronizeWindowsEvent event) {

```

Comment:
I am confused BookmarksChanged should cause a screen refresh not a windows sync.

Reply:
Perhaps better name would be something like "ScreenRefreshEvent"?